tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -32,7 +32,6 @@ public static HashSet<string> s_scope
         public const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
         public const string ClientCredentialAudience = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0";
         public const string AutomationTestThumbprint = "378938210C976692D7F523B8C4FFBB645D17CE92";
-        public const string RSATestCertThumbprint = "A7F68F87199F412451BF0FB364A3358A949D2CA8";

```

Comment:
@trwalke - do you remember where this was used?

Reply:
it is here. I think this cert is used for our lab access
https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/asset/Microsoft_Azure_KeyVault/Certificate/https://buildautomation.vault.azure.net/certificates/RSATestCertDotNet

Reply:
Looks like the tests have all passed though. We probably do not need it in code.

Reply:
Updated LabAccessThumbPrint with AutomationTestThumbprint value and Lab access is working as expected, and all tests are passing